docs/_articles/en/codebuilder/cb-tips.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Useful Tips for Using Code Builder
+lang: en
+---
+
+## Install Extensions from Open VSX Registry
+
+Development experience in Code Builder can be greatly enhanced by installing additional extensions to your workspace. [Open VSX](open-vsx.org) is an Eclipse open-source project and alternative to the Visual Studio Marketplace. Extensions available here can be installed in Code Builder. You can browse and install these extensions from within VS Code. Bring up the Extensions view by clicking on the Extensions icon in the Activity Bar or the **View: Extensions** command (⇧⌘X).
+
+## Reset your Code Builder Environment
+
+Reset your Code Builder environment to start building from scratch. Resetting deletes all our data so you have a clean Salesforce DX to work with. To reset your project, click the dropdown next to the launch button and select **Reset Environment**.
```

Comment:
@sbudhirajadoc - Deleting all data sounds scary and might lead folks to believe it deletes data from their org.  Suggested edit:

Reset your Code Builder environment to start building with a fresh environment. Resetting deletes everything stored in your Code Builder environment all our data so you have a clean space to work with. To reset your Code Builder environment, click the dropdown next to the launch button and select **Reset Environment**.

Reply:
updated. thanks!